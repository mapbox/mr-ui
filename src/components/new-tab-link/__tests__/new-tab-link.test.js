import React from 'react';
import { shallow } from 'enzyme';
import { testCases } from './new-tab-link-test-cases';

describe('NewTabLink', () => {
  test(testCases.basic.description, () => {
    const wrapper = shallow(
      React.createElement(testCases.basic.component, testCases.basic.props)
    );
    expect(wrapper).toMatchSnapshot();
  });

  test(testCases.styling.description, () => {
    const wrapper = shallow(
      React.createElement(testCases.styling.component, testCases.styling.props)
    );
    expect(wrapper).toMatchSnapshot();
  });
});
