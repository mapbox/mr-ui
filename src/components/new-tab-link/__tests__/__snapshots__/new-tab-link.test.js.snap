// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewTabLink basic 1`] = `
<a
  href=""
  rel="noopener"
  target="_blank"
>
  Hello explorer!
</a>
`;

exports[`NewTabLink styling 1`] = `
<a
  className="link link--pink"
  href="http://www.mapbox.com"
  rel="noopener"
  target="_blank"
>
  Hello explorer!
</a>
`;
